__tests__/app.test.js

```diff
@@ -50,6 +50,31 @@ describe("/api/topics", () => {
 });
 
 describe("/api/articles", () => {
+  describe("GET", () => {
+    test("200 + returns all articles", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+          const articles = body.articles;
+          articles.forEach((article) => {
+            expect(article).toEqual(
+              expect.objectContaining({
+                article_id: expect.any(Number),
+                title: expect.any(String),
+                topic: expect.any(String),
+                author: expect.any(String),
+                body: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+                article_img_url: expect.any(String),
+              })
```

Comment:
The ticket says that we don't want body to come back, whilst we also want to create 'comment_count'. Take a look at my comment in your model for more info 🙂